### PR TITLE
Add Series float rounding methods: round/2, floor/1 and ceil/1

### PR DESCRIPTION
**Pull request by sasikumar87**

### CODE REVIEW

File: native/explorer/src/expressions.rs [hunk 1]
```diff
@@ -598,3 +598,21 @@ pub fn expr_trim_trailing(expr: ExExpr) -> ExExpr {
     let expr: Expr = expr.resource.0.clone();
     ExExpr::new(expr.str().rstrip(None))
 }
+
+#[rustler::nif]
+pub fn expr_round(expr: ExExpr, decimals: f64) -> ExExpr {
```
Comment (philss) [inline native/explorer/src/expressions.rs:603]: I think you could use an integer for decimals. Have you tried `u32` here?

File: native/explorer/src/series.rs [hunk 2]
```diff
@@ -879,3 +879,21 @@ pub fn s_trim_trailing(data: ExSeries) -> Result<ExSeries, ExplorerError> {
     // There are no eager strip functions.
     Ok(ExSeries::new(s1.utf8()?.replace(r#"[ \s]+$"#, "")?.into()))
 }
+
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_round(data: ExSeries, decimals: i64) -> Result<ExSeries, ExplorerError> {
```
Comment (philss) [inline native/explorer/src/series.rs:884]: Same here about the integer.

**Review by philss — COMMENTED** [review]
Cool! Thanks! :)

**Review by josevalim — APPROVED** [review]
LGTM besides @philss’ comments.

**Review by philss — APPROVED** [review]
:shipit:

**Comment by philss** [comment]
@sasikumar87 thank you! :purple_heart:

**Pull request merged**